docs/jemalloc_usage.md

```diff
@@ -0,0 +1,29 @@
+# JEMalloc usage
+FEX-Emu uses two different jemalloc heap allocators at once.
+- jemalloc: The primary heap allocator
+- jemalloc_glibc: The second heap allocator
+
+## jemalloc - primary heap allocator
+This heap allocator is the primary allocator that FEX-Emu uses. All of FEXCore's `fextl::` namespaced objects will allocate memory through this
+allocator. This allocator has its usage of mmap and munmap overridden to use FEXCore's internal VMA region allocator.
```

Comment:
Phrasing is a bit roundabout here. I'd describe what this actually does first, then where it's used (since the section header already mentions it's the primary allocator):

```suggestion
This allocator overrides `mmap` and `munmap` by forwarding them to FEXCore's internal VMA region allocator.

All of FEXCore's `fextl::` namespaced objects allocate memory with this method.
```

Reply:
Changed phrasing to yours.